README.md

```diff
@@ -44,12 +44,34 @@ cargo build --release
 
 **Pura** can be used as a standalone container runtime like **runc** with the OCI compliant commands:
 
+Make this to clean the way:
+```sh
+set ID=example
+[sudo] unlink /tmp/pura/${ID}/init.sock
+[sudo] unlink /tmp/pura/${ID}/container.sock
 ```
+
+On another terminal:
+```sh
+[sudo] nc -vklU /tmp/${ID}.sock # on another terminal
+```
+
+On the config.json
+```sh
+...
+"root": {
+  "path": "/path/to/rootfs"
+}
```

Comment:
Actually, the `root` section on Linux is required, so we don't need to specify it in the documentation.

Reply:
You need to specific the complete path to `rootfs` directory

Reply:
Yup, got it this morning in my tests.